pyvortex/src/io.rs

```diff
@@ -61,16 +61,18 @@ use crate::{PyArray, TOKIO_RUNTIME};
 ///
 /// >>> c = vortex.io.read_path("a.vortex", projection = ["age"])
 /// >>> c.to_arrow_array()
-/// <pyarrow.lib.StructArray object at ...>
-/// -- is_valid: all not null
-/// -- child 0 type: int64
-///   [
-///     25,
-///     31,
-///     null,
-///     57,
-///     null
-///   ]
+/// <pyarrow.lib.ChunkedArray object at ...>
+/// [
+///   -- is_valid: all not null
+///   -- child 0 type: int64
+///     [
+///       25,
+///       31,
+///       null,
+///       57,
+///       null
+///     ]
+/// ]
```

Comment:
My gut: `write(struct, f); x = read(f), assert struct == x` should not fail. I guess this is the V2 layout stuff forcibly converting every array into a chunked one?